### ISSUE TITLE
Content creators monetization question

### ISSUE THREAD

**Issue by mankota** [WWBN/AVideo#9514]
Daniel,
How can the content creators be able to monetize their videos through ads? is there a dashboard where they can see their earnings?

**Comment by DanielnetoDotCom** [comment]
Yes, content creators can monetize their videos through ads using the **AD_Overlay Plugin** or **video ads (VAST/VPAID)** if the site administrator has enabled these features. This allows creators to insert their own ad codes over their content or use the site's advertising system.

### Monetization via Ads for Content Creators:
- **AD_Overlay Plugin**: This plugin allows individual content creators to insert their own ad codes. For example, if a creator has their own Google AdSense account, they can display ads on their videos and earn revenue from the ad impressions.
- **Video Ads (VAST/VPAID)**: If the site owner has implemented video ads through systems like GoogleAds_IMA, creators may also benefit from ad revenue based on video views and ad impressions.

### Dashboard for Earnings:
Currently, content creators can monitor their earnings via the **YPTWallet Plugin**. The YPTWallet provides a dashboard where creators can track their balance from various revenue streams, including:
- **Subscriptions**
- **PayPerView purchases**
- **Donations**
- **Ad revenue (if allowed by the site owner)**

Through the **YPTWallet** dashboard, creators can see how much they have earned and, if enabled, request withdrawals based on the site's policies.

### Summary:
- Creators **can monetize through ads**, either by adding their own ads or benefiting from the site's ad system.
- The **YPTWallet Plugin** acts as a dashboard for creators to view their earnings from various sources, including ads, subscriptions, and donations.

For more details, you can check out the related [AD_Overlay Plugin documentation](https://github.com/WWBN/AVideo/wiki/AD_Overlay-Plugin) and [YPTWallet Plugin documentation](https://github.com/WWBN/AVideo/wiki/YPTWallet-Usage).
